networks/rpc/server.go

```diff
@@ -171,6 +171,7 @@ func (s *Server) serveRequest(ctx context.Context, codec ServerCodec, singleShot
 	}
 	s.codecsMu.Lock()
 	if atomic.LoadInt32(&s.run) != 1 { // server stopped
+		rpcErrorResponsesCounter.Inc(1)
```

Comment:
This makes `rpcErrorResponsesCounter` more than `rpcTotalRequestsCounter`. Is it appropriate to raise `rpcErrorResponsesCounter` instead?

Reply:
Server is not running and it does not check requests. I removed it. Thanks.